Sources/Auth/Apple/AppleAuthProvider.swift

```diff
@@ -96,9 +96,21 @@ extension AppleAuthProvider: ASAuthorizationControllerDelegate {
       // store userId for later
       Self.storage.set(credential.user, forKey: Self.userIdStorageKey)
       
+      // parse email and it's metadata
```

Comment:
These and the next two comments feel a bit unnecessary.

Reply:
I think better more than less 🤷‍♂️